src/OpenSage.Game/Terrain/Roads/RoadConstants.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace OpenSage.Terrain.Roads
+{
+    public static class RoadConstants
+    {
+        // TODO figure out exact threshold
+        public const float MinCurveAngle = 0.47f;
```

Comment:
I guess they used some degree and converted it to radians. The next degree would be 27° with 0.47123889804.